packages/flame/HOW_TO_ADD_ICONS.md

```diff
@@ -12,7 +12,7 @@ yarn bootstrap
 This script will generate the React components and the Icon sprite.
 
 ```sh
-yarn run-svgo
+yarn run svgo
```

Comment:
https://github.com/lightspeed/flame/blob/next/packages/flame/package.json#L16 current one is `run-svgo`

Reply:
I ran in the wrong folder 🤦